src/views/datasets/buoy-telemetry/dashboard.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <LineChartDashboard :query="query" :exclude-nulls="true">
+    <template #summary-heatmap>
+      <StationHeatmap
+        :summary="store.summary"
+        :variables="store.variables"
+        x-title="Date"
+        x-unit="day"
+      />
+    </template>
+  </LineChartDashboard>
+</template>
+
+<script setup lang="ts">
+import { inject, provide, watch } from "vue";
+import { useRoute } from "vue-router";
+
+import LineChartDashboard from "@/components/buoy/LineChartDashboard.vue";
+import StationHeatmap from "@/components/buoy/StationHeatmap.vue";
+
+import { BuoyStore } from "../../../store/buoy";
+const store = inject("store") as BuoyStore;
+const route = useRoute();
+import { useQuery } from "../../../composables/useQuery";
+const { query, updateQuery } = useQuery(store, route.path);
+
+updateQuery(route.query, route.path);
+watch(
+  () => route.query,
+  (val) => updateQuery(val, route.path)
+);
+
+// set up the comparison dataset
+import { buoyStores } from "../../../store/buoy";
+const compareStore = buoyStores["osom"].useStore();
```

Comment:
hmmm why is the "osom" store used in here?

Reply:
I honestly don't know why this is needed. I think Mary set it up for something with the line charts. Somewhere in that process, a comparison data set is needed. I can set up some pairing with @eldu to see if we can figure out what it does.

Reply:
Comparison when osom is available.

Reply:
It looks like the plankton pages is also using this. Looks like this is the store that is being compared to which makes since it's the Ocean Station Ocean Model.

We can take a deeper dive when i'm back in June !